src/ralph/accounts/admin.py

```diff
@@ -85,6 +86,12 @@ def url(self, item):
         )
     url.title = _('Link')
 
+    def buyout_date(self, item):
+        return Asset.get_buyout_date(
+            item['invoice_date'], item['depreciation_rate']
```

Comment:
no - it's yearly drop on value; so depreciation_rate = 25 means, that every year asset will drop by 25% on it's original value, reaching value 0 (depreciated) after 4 years.